README.md

```diff
@@ -48,29 +38,28 @@ Preferably, please cite the following paper:
   - Maxime Baudette, Marcelo Castro, Tin Rabuzin, Jan Lavenius, Tetiana Bogodorova, Luigi Vanfretti, OpenIPSL: Open-Instance Power System Library — Update 1.5 to “iTesla Power Systems Library (iPSL): A Modelica library for phasor time-domain simulations”, SoftwareX, Volume 7, January–June 2018, Pages 34-36, ISSN 2352-7110, https://doi.org/10.1016/j.softx.2018.01.002
     - Link to Publisher's Website: [SoftwareX](https://www.sciencedirect.com/science/article/pii/S2352711018300050)
 
-You are also welcome to submit your contributions as stated above.
+You are also welcome to submit your contributions as stated below.
 
-## Copyright and License Information:
+## Contributing:
+Contributions to the library are welcome and can be submitted in the form of pull requests to this repository. 
 
-The OpenIPSL is released under the same license as former iPSL project.
-All changes applied in this fork have the following copyright statement.
+Please consult the [contribution guidelines](http://openipsl.readthedocs.io/en/latest/community/contributing.html). More information about our contributors can be found over [here](http://openipsl.readthedocs.io/en/latest/community/community-home.html). 
```

Comment:
No link to readthedocs. Better create a new file following: https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors and link to that one.